routes/pointofinterest/route.js

```diff
@@ -41,17 +46,74 @@ router.use((req, res, next) => {
   next();
 });
 
+// Get the id of the poi created previously
+let resultPicture;
+
 router.post('/', (req, res) => {
-  const formData = req.body;
-  connection.query(createNewPoi, [formData], (err) => {
+  const formData = {
+    name: req.body.name,
+    latitude: req.body.latitude,
+    longitude: req.body.longitude,
+  };
+  const grades = {
+    global_grade: req.body.global_grade,
+    accessibility: req.body.accessibility,
+    condition: req.body.condition,
+    functional: req.body.functional,
+  };
+  const authorName = req.body.author_id;
+  const { keyword } = req.body;
+
+
+  connection.query(createNewPoi, [formData, authorName, resultPicture], (err, result) => {
     if (err) {
-      res.status(500).send(`Erreur lors de la récupération des données : ${err}`);
+      res.status(500).send(`Erreur lors de la création du point d'intéret : ${err}`);
     } else {
-      res.sendStatus(200);
+      // Get the id of the poi created previously
+      const resultId = { poi_id: result.insertId };
+      connection.query(linkNewlyCreatedPoiWithKeyword, [keyword, resultId], (error) => {
+        if (error) {
+          res.status(500).send(`Erreur lors de l'ajout du thème : ${error}`);
+        } else {
+          connection.query(insertGradesNewPoi, [grades, authorName, resultId.poi_id], (e) => {
```

Comment:
not ```e```for ```error``` error_grades would be better for exemple (and if you change this one like this, change the other)

Reply:
changed but in camelCase to follow linter instructions